src/Spd.Resource.Repository/LicApp/LicAppRepository.cs

```diff
@@ -24,14 +24,19 @@ public async Task<LicenceApplicationCmdResp> CommitLicenceApplicationAsync(Guid
         if (app == null)
             throw new ApiException(HttpStatusCode.BadRequest, "Invalid ApplicationId");
 
-        if (status == ApplicationStatusEnum.Submitted)
-        {
-            app.statuscode = (int)ApplicationStatusOptionSet.Submitted;
-            app.statecode = DynamicsConstants.StateCode_Inactive;
-        }
-        else
+        app.statuscode = (int)Enum.Parse<ApplicationStatusOptionSet>(status.ToString());
+
+        switch (status)
         {
-            app.statuscode = (int)Enum.Parse<ApplicationStatusOptionSet>(status.ToString());
+            case ApplicationStatusEnum.Submitted:
+                app.statecode = DynamicsConstants.StateCode_Inactive;
+                break;
+            case ApplicationStatusEnum.ApplicantVerification:
+                app.spd_identityconfirmed = false;
```

Comment:
We should not just save the spd_identityconfirmed when user do the commit. It should be saved once the application just got created.

Reply:
the old code should not change.

Reply:
now handling spd_identityconfirmed in the repositories.